### DISCUSSION TITLE
[pem] Support creating JWT-signing keys (.p8 files) in pem

### DISCUSSION THREAD

**Discussion by jrbeilke** [fastlane/fastlane#18628] [Ideas]
Maybe need to re-open #7848 and #9256 since it was auto-closed

@snatchev looks like this functionality was added to spaceship but never made it into pem unless I'm missing something?

**Comment by max-ott** [comment]
As there was not really any huge demand for this feature it never ended up in PEM. You can still use it from what I read by calling the code directly through Spaceship.

```
key = Spaceship::Portal::Key.create(name: 'My New Push Key', apns: true)
File.write('push-jwt.p8', key.download)

keys = Spaceship::Portal::Key.all
keys.each do |key|
  key.revoke!
end
```
